.github/workflows/on-droneci-complete.yml

```
@@ -1,19 +1,86 @@
-# This is an experiment with triggering a GitHub worfklow AFTER Drone CI completes.
+# After DroneCI completes, we need to close the Applitools Eyes batch (and potentially
+# do other GH actions tasks in the future).
 #
-# Delete this file if the date is after July 2025 and this comment is not removed,
-# or if https://github.com/code-dot-org/code-dot-org/pull/65111 is closed.
-#
-# This file is aprt of https://github.com/code-dot-org/code-dot-org/pull/65111, but
-# we cannot test the GitHub action until this workflow file exists and is merged
-# into `staging`. We override the contents of the file in the PR.
+# This workflow is triggered by an HTTP post from the `on-droneci-complete` job in .drone.yml
 name: on-droneci-complete
 
 on:
   workflow_dispatch:
+    inputs:
+      github_sha:
+        description: "equivalent to github.sha in an ordinary workflow"
+        required: true
+      drone_build_status:
+        required: true
+        type: choice
+        options:
+          - success
+          - failure
+
+permissions:
+  statuses: write
 
 jobs:
-  replaceme:
-    if: false
-    runs-on: ubuntu-latest
+  # We use applitools eyes from: drone and frontend-ci.yml. We need to close the eyes batch
+  # after these are done. Drone is so slow that we assume Drone will be the last to finish,
+  # but we check and throw an error if frontend-ci.yml is still running when drone completes.
+  complete-applitools-eyes-batch:
+    runs-on: ubuntu-24.04
     steps:
-      - run: true
+      - run: |
+          echo "Triggered by Drone, on commit: ${{ inputs.github_sha }} on branch ${{ github.ref }}"
+          echo "This job should succeed"
+          true
+
+      - name: Assert other eyes-using workflows are already complete
+        uses: actions/github-script@v7
```

After reading about the recent [Github Action supply chain attack](https://nvd.nist.gov/vuln/detail/cve-2025-30066), I'm more hesitant to use version tags since Github Actions aren't locked like npm packages are. Github Action's hardening guide [recommends](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) to use the commit sha instead and dependabot can update the commit hash too.

Although, this is an official action so the attack vector is significantly less likely to happen.

```suggestion
        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
```